up42/processing.py

```diff
@@ -71,6 +74,8 @@ class Job:
     created: datetime.datetime
     updated: datetime.datetime
     collection_url: Optional[str] = None
+    errors: Optional[List[ValidationError]] = None
+    credits: Optional[int] = None
```

Comment:
I see in the job template we create a class for comparing JobTemplate costs. Why did we choose to just use credits in this case?. `hold_id` and `consumption_id` are not explicit properties with this approach. If that is intended then OK with me.